lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -116,6 +116,99 @@ class CategoryRenderer {
   }
 
   /**
+   * @param {!ReportRenderer.AuditJSON} audit
+   * @param {number} scale
+   * @return {!Element}
+   */
+  _renderPerfHintAudit(audit, scale) {
+    const extendedInfo = /** @type {!CategoryRenderer.PerfHintExtendedInfo}
```

Comment:
could this be in `details`? It would be nice to not split report rendering info across `details` and `extendedInfo` if we don't have to